Nodejs lerna version mismatch

https://travis-ci.org/imsky/PocketEuler/builds/158685369

```
> @ postinstall /home/travis/remark
> lerna bootstrap
Lerna v2.0.0-beta.28
Independent Versioning Mode
Lerna version mismatch: The current version of lerna is 2.0.0-beta.28, but the Lerna version in `lerna.json` is 2.0.0-beta.26. You can either run `lerna init` again or install `lerna@2.0.0-beta.26`.
npm ERR! Linux 3.13.0-92-generic
npm ERR! argv "/home/travis/.nvm/v0.11.16/bin/node" "/home/travis/.nvm/v0.11.16/bin/npm" "install" "--production"
npm ERR! node v0.11.16
npm ERR! npm  v2.3.0
npm ERR! code ELIFECYCLE
npm ERR! @ postinstall: `lerna bootstrap`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ postinstall script 'lerna bootstrap'.
npm ERR! This is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     lerna bootstrap
npm ERR! You can get their info via:
npm ERR!     npm owner ls 
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/remark/npm-debug.log
The command "npm install --production" failed and exited with 1 during .
```

And when that is fixed, other problems occur.
https://travis-ci.org/imsky/PocketEuler/builds/158687076

```bash
$ node $HOME/remark/packages/remark-cli/cli.js --config=./.remarkrc.js --frail --no-stdout -u remark-lint README.md
README.md
  1:1  error  Error: Cannot find module `remark-lint`
    at /home/travis/remark/packages/remark-cli/node_modules/unified-args/node_modules/unified-engine/lib/file-pipeline/configure.js:68:19
    at /home/travis/remark/packages/remark-cli/node_modules/unified-args/node_modules/unified-engine/lib/configuration.js:444:7
    at Array.forEach (native)
    at /home/travis/remark/packages/remark-cli/node_modules/unified-args/node_modules/unified-engine/lib/configuration.js:443:13
    at /home/travis/remark/packages/remark-cli/node_modules/unified-args/node_modules/unified-engine/lib/configuration.js:327:5
    at /home/travis/remark/packages/remark-cli/node_modules/unified-args/node_modules/unified-engine/node_modules/vfile-find-up/index.js:135:9
    at FSReqWrap.oncomplete (fs.js:99:15)
✖ 1 error
The command "node $HOME/remark/packages/remark-cli/cli.js --config=./.remarkrc.js --frail --no-stdout -u remark-lint README.md" exited with 1.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,17 +3,9 @@ node_js: 0.11
 install:
   - gem install mdl
   - npm install -g markdownlint-cli
-
-  # Obtain development version of remark
-  - git clone --depth 1 https://github.com/wooorm/remark $HOME/remark
-  - cd $HOME/remark
-  - git show --oneline -s
-  - npm install lerna
-  - npm install --production
-  - npm install remark-validate-links remark-lint
-  - cd -
+  - npm install -g remark-cli remark-validate-links remark-lint
 script:
   - mdl -c ./.mdlrc README.md
   - markdownlint -c ./.markdownlint.json README.md
-  - node $HOME/remark/packages/remark-cli/cli.js --config=./.remarkrc.js --frail --no-stdout -u remark-lint README.md
-  - node $HOME/remark/packages/remark-cli/cli.js --config=./.remarkrc.js --frail --no-stdout -u remark-validate-links README.md
+  - remark --config=./.remarkrc.js --frail --no-stdout -u remark-lint README.md
+  - remark --config=./.remarkrc.js --frail --no-stdout -u remark-validate-links README.md

